docs/examples/junit4/generic/src/test/java/generic/ExecTest.java

```diff
@@ -26,8 +26,8 @@ public void testSimpleExec() throws IOException, InterruptedException {
         Container.ExecResult lsResult = container.execInContainer("ls", "-al", "/");
         String stdout = lsResult.getStdout();
         int exitCode = lsResult.getExitCode();
-        assertTrue(stdout.contains("somefile.txt"));
-        assertTrue(exitCode == 0);
+        assertThat(stdout.contains("somefile.txt")).isTrue();
+        assertThat(exitCode == 0).isTrue();
```

Comment:
```suggestion
        assertThat(stdout).contains(somefile.txt);
        assertThat(exitCode).isEqualTo(0);
```

Ok, in this case, the original asserts were not great :D